CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All changes made on any release of this project should be commented on high leve
 Document model based on [Semantic Versioning](http://semver.org/).
 Examples of how to use this _markdown_ cand be found here [Keep a CHANGELOG](http://keepachangelog.com/).
 
+## [0.11.0](https://github.com/stone-payments/kong-plugin-template-transformer/tree/v0.11.0) - 2019-05-16
+### Fixed
+- Special characters scaped in body_filter.
```

Comment:
If this is a fix the new version should be a patch, not a Minor.

Reply:
@andremacdowell we normally release fixes as minor when they are not hotfixes. since the code has already been pushed to develop, we're considering it a non-critical bugfix.

Reply:
You tipically want your users to download bugfixes as soon as possible, even if they are not critical. A user will look differently for a package that has a new minor version (tipically new feature) from a package that has a new patch version (security and performance features or bugs)